public_src/components/sidebar/validation-browser.component.html

```diff
@@ -1,37 +1,63 @@
-    <ng-container *ngIf="!(errorCorrectionMode$ | async) ||(errorCorrectionMode$ | async) === 'menu' ">
-      <p> {{'Zoom into the area where you want to run the validation.' | translate}}</p>
-      <button class="btn btn-default" (click)="startValidation()"> {{'Find Errors' | translate }}</button>
-    </ng-container>
-    <ng-container *ngIf="(switchMode$ | async)">
-      <button class="btn btn-default" type="submit" id='prevLoc' (click)="previousLocation()"><i
-        class="fa fa-arrow-left" aria-hidden="true"></i></button>
-      <button class="btn btn-default" type="submit" id='nextLoc' (click)="nextLocation()"><i class="fa fa-arrow-right"
-                                                                                             aria-hidden="true"></i>
-      </button>
-      <button class="btn btn-default pull-right" type="submit" id='quit' (click)="quit()">{{'Quit' | translate }}
-      </button>
+    <ng-container *ngIf="(errorCorrectionMode$ | async) as errorCorrectMode">
+
+      <ng-container
+        *ngIf="view('find-errors-option')">
+        <p> {{'Zoom into the area where you want to run the validation.' | translate}}</p>
+        <button class="btn btn-default" (click)="startValidation()"> {{'Find Errors' | translate }}</button>
+      </ng-container>
+
+      <ng-container *ngIf="(switchMode$ | async)">
+        <button class="btn btn-default" type="submit" id='prevLoc' (click)="previousLocation()"><i
+          class="fa fa-arrow-left" aria-hidden="true"></i></button>
+        <button class="btn btn-default" type="submit" id='nextLoc' (click)="nextLocation()"><i class="fa fa-arrow-right"
+                                                                                               aria-hidden="true"></i>
+        </button>
+        <button class="btn btn-default pull-right" type="submit" id='quit' (click)="quit()">{{'Quit' | translate }}
+        </button>
+      </ng-container>
+
     </ng-container>
   </div>
+  <ng-container>
+
+    <ng-container *ngIf="view('menu')">
 
-  <div class="content panel panel-body panel-default scrollable-panel"
-       *ngIf="(errorCorrectionMode$ | async) === 'menu' || (errorCorrectionMode$ | async) === 'missing name tag' || (errorCorrectionMode$ | async) === 'missing ref tag'">
-    <ng-container *ngIf="(errorCorrectionMode$ | async) === 'menu'">
       <ul class="list-group">
-        <li class="list-group-item" *ngIf="nameErrorsO"> {{nameErrorsO.length}} {{'missing name tags' | translate }}
-          <button (click)="startNameCorrection()">{{'Start Correction' | translate }}</button>
+
+        <li class="list-group-item clearfix" *ngIf="view('name-errors-menu-item')">
+          <ng-container *ngIf="nameErrorsObj.length !==0">
+            {{nameErrorsObj.length}} {{'name tag errors' | translate }}
+            <button class="btn btn-default pull-right" (click)="startNameCorrection()">{{'Start Correction' | translate
+              }}
```

Comment:
I would prefer to see }} on the same line together with the rest of inner content